src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.health = health;
+    this.name = name;
+    this.hidden = hidden;
```

Comment:
The 'hidden' property is being initialized for all animals, but it should only be relevant for instances of Herbivore. You should remove this property from the Animal class and ensure it is only used within the Herbivore class.